cmd/test-go-table-01/main.go

```diff
@@ -75,13 +75,14 @@ func main() {
 	}
 
 	// write one packet to the packet table
-	if err = table.Append(&particles[0]); err != nil {
+	if err = table.Append(particles[0]); err != nil {
 		panic(fmt.Errorf("Append failed with single packet: %s", err))
 	}
 
 	// write several packets
-	parts := particles[1:]
-	if err = table.Append(&parts); err != nil {
+	if err = table.Append(particles[1], particles[2], particles[3],
```

Comment:
`err = Append(particles[1:]...)`

Also, I'm not a huge fan of unnecessary wrapping of assignments in if constructs, but this was here, so I guess it can stay.

Reply:
Results in `./main.go:83: cannot use particles[1] (type particle) as type []interface {} in argument to table.Append`

Reply:
Yes, sorry. Can you put this in a loop then?

Reply:
Sure!

Reply:
But actually: that would defeat the purpose of this test, wont you agree? This is supposed to test to append multiple `particle` at once as opposed to a test like

```
for _, part := range particles {
	if err = table.Append(part); err != nil {
		panic(err)
	}
}
```

which would append one at a time.

Reply:
Yes, I'd like to actually test that code path :)

Reply:
In that case, can you do a loop to load a `[]interface{}` and then use `Append(p...)`, and when this becomes a test rather than a main, it shouldn't be an example. It does seem an odd thing to test though - what is the rationale for it?

Reply:
I don't exactly know about the rationale here, as I haven't written those files. Just fixed them in order to make them work :)

Reply:
So I suggest that these things are fixed when https://github.com/gonum/hdf5/issues/22 is tackled.